### PR TITLE
Add ci test for causal compact mask bert

### PR DESCRIPTION
**Pull request by jh619lee**
## 문제상황
- 이전 PR49의 [리뷰](https://github.com/furiosa-ai/inference-compression/pull/49#discussion_r1647506182)에 대한 수정 필요.
- mlperf submission 옵션 중 하나인 compact_causal_mask bert에 대해서도 동치 확인을 수행할 수 있는 ci test 필요

## 목적(해결방향)
- 이전 PR49 리뷰 반영
- compact_causal_mask bert ci test 추가

## 구현 설명 Abstract
- 이전 PR49 리뷰를 반영하여 make_dataloader의 data source specific한 부분 제거.
- ci test 간 통일성을 위해 run_ci.py 리네이밍 진행.
- compact_causal_mask bert ci test 추가.

## 구현 설명 Detail (ex. 추가된 argument)
- language/bert/quantization/calib_dataloader/calib_dataloader.py (이전 PR 리뷰 반영 @jeongin-yun)
    - make_dataloader에 들어오는 data의 kwargs를 bert model kwargs와 일치시켜, makde_dataloader의 data source-specific한 부분 제거.
    
- language/bert/run_ci.py
    - test model specific한 변수들을 제거 하고,  (golden-comparison) 형태로 리네이밍.
    - run_ci_compact_causal_mask.py와 같이 독립적인 ci test들 간의 통일성 유지를 위함.
    
- language/bert/run_ci_compact_causal_mask.py
    - attention mask의 차이로 인해 calibration을 각각 수행하면 qparam이 다르게 얻어지므로, compact causal mask bert에 대한 동치 확인은 아래와 같이 구성 
        - hf bert vs mlperf submission bert 동치를 확인하였던 환경에서 qparam 생성-> 두 qparam이 동일하게 얻어졌는지 확인
        - compact causal mask bert 모델을 위에서 얻은 qparam으로 qlv4 변환 후, forwarding (별도의 qparam matching 과정 없이 동작)
        - valid seq 영역에 대하여 hf bert vs compact causal mask bert logits 비교 

    
## test 통과 내역 (CI 통과내역과 어떤 machine (GPU pod or NPU machien or local)에서 테스트했는지)
- [ ] local machine with 3090
- [X] GPU pod
    - python run_ci.py @language/bert 
    - python run_ci_compact_causal_mask.py @language/bert 
    
- [ ] warboy pod

## TODO (ex. 후속PR예고)
-

### CODE REVIEW

**Comment by BeomGeunCho** [comment]
mlperf_submission 모델 calibration (10개) evaluation (10개) 로 확인해본 결과
현재 main 과 bert_ci_test branch 에서 동일한 f1 score 가 나오는 걸 확인했습니다.

**Review by BeomGeunCho — APPROVED** [review]
LGTM!
